gulpfile.js

```diff
@@ -110,13 +110,20 @@ function createWebpackConfig(defines, output) {
     module: {
       loaders: [
         {
+          test: /(\bweb\b).*?\.js$/,
```

Comment:
*Really sorry to keep asking, but I'm still not sure that I understand the meaning of this RegExp.*

Don't you want to use e.g. `/\.js$/` instead, such that this is *not* limited to just the `web` folder?